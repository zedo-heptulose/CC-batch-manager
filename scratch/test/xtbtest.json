{"SP_energy_au": -3.769421097051, "bandgap_ev": null, "completion_success": true}