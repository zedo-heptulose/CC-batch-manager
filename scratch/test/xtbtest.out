      -----------------------------------------------------------      
     |                   =====================                   |     
     |                           x T B                           |     
     |                   =====================                   |     
     |                         S. Grimme                         |     
     |          Mulliken Center for Theoretical Chemistry        |     
     |                    University of Bonn                     |     
      -----------------------------------------------------------      

   * xtb version 6.6.1 (8d0f1dd) compiled by 'conda@1efc2f54142f' on 2023-08-01

   xtb is free software: you can redistribute it and/or modify it under
   the terms of the GNU Lesser General Public License as published by
   the Free Software Foundation, either version 3 of the License, or
   (at your option) any later version.
   
   xtb is distributed in the hope that it will be useful,
   but WITHOUT ANY WARRANTY; without even the implied warranty of
   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
   GNU Lesser General Public License for more details.
   
   Cite this work as:
   * C. Bannwarth, E. Caldeweyher, S. Ehlert, A. Hansen, P. Pracht,
     J. Seibert, S. Spicher, S. Grimme, WIREs Comput. Mol. Sci., 2020, 11,
     e01493. DOI: 10.1002/wcms.1493
   
   for GFN2-xTB:
   * C. Bannwarth, S. Ehlert and S. Grimme., J. Chem. Theory Comput., 2019,
     15, 1652-1671. DOI: 10.1021/acs.jctc.8b01176
   for GFN1-xTB:
   * S. Grimme, C. Bannwarth, P. Shushkov, J. Chem. Theory Comput., 2017,
     13, 1989-2009. DOI: 10.1021/acs.jctc.7b00118
   for GFN0-xTB:
   * P. Pracht, E. Caldeweyher, S. Ehlert, S. Grimme, ChemRxiv, 2019, preprint.
     DOI: 10.26434/chemrxiv.8326202.v1
   for GFN-FF:
   * S. Spicher and S. Grimme, Angew. Chem. Int. Ed., 2020, 59, 15665-15673.
     DOI: 10.1002/anie.202004239
   
   for ALPB and GBSA implicit solvation:
   * S. Ehlert, M. Stahn, S. Spicher, S. Grimme, J. Chem. Theory Comput.,
     2021, 17, 4250-4261. DOI: 10.1021/acs.jctc.1c00471
   
   for DFT-D4:
   * E. Caldeweyher, C. Bannwarth and S. Grimme, J. Chem. Phys., 2017,
     147, 034112. DOI: 10.1063/1.4993215
   * E. Caldeweyher, S. Ehlert, A. Hansen, H. Neugebauer, S. Spicher,
     C. Bannwarth and S. Grimme, J. Chem. Phys., 2019, 150, 154122.
     DOI: 10.1063/1.5090222
   * E. Caldeweyher, J.-M. Mewes, S. Ehlert and S. Grimme, Phys. Chem. Chem. Phys.
     2020, 22, 8499-8512. DOI: 10.1039/D0CP00502A
   
   for sTDA-xTB:
   * S. Grimme and C. Bannwarth, J. Chem. Phys., 2016, 145, 054103.
     DOI: 10.1063/1.4959605
   
   in the mass-spec context:
   * V. Asgeirsson, C. Bauer and S. Grimme, Chem. Sci., 2017, 8, 4879.
     DOI: 10.1039/c7sc00601b
   * J. Koopman and S. Grimme, ACS Omega 2019, 4, 12, 15120-15133.
     DOI: 10.1021/acsomega.9b02011
   
   for metadynamics refer to:
   * S. Grimme, J. Chem. Theory Comput., 2019, 155, 2847-2862
     DOI: 10.1021/acs.jctc.9b00143
   
   for SPH calculations refer to:
   * S. Spicher and S. Grimme, J. Chem. Theory Comput., 2021, 17, 1701-1714
     DOI: 10.1021/acs.jctc.0c01306
   
   with help from (in alphabetical order)
   P. Atkinson, C. Bannwarth, F. Bohle, G. Brandenburg, E. Caldeweyher
   M. Checinski, S. Dohm, S. Ehlert, S. Ehrlich, I. Gerasimov, C. Hölzer
   A. Katbashev, J. Koopman, C. Lavigne, S. Lehtola, F. März, M. Müller,
   F. Musil, H. Neugebauer, J. Pisarek, C. Plett, P. Pracht, F. Pultar,
   J. Seibert, P. Shushkov, S. Spicher, M. Stahn, M. Steiner, T. Strunk,
   J. Stückrath, T. Rose, and J. Unsleber
   
 * started run on 2024/10/14 at 13:29:59.179     

           -------------------------------------------------
          |                Calculation Setup                |
           -------------------------------------------------

          program call               : xtb test.xyz --gfn 2 --chrg 0 --uhf 0 --opt --cma
          hostname                   : hpc-i32-2
          coordinate file            : test.xyz
          omp threads                :                     4

   ID    Z sym.   atoms
    1    8 O      1

           -------------------------------------------------
          |                 G F N 2 - x T B                 |
           -------------------------------------------------

        Reference                      10.1021/acs.jctc.8b01176
      * Hamiltonian:
        H0-scaling (s, p, d)           1.850000    2.230000    2.230000
        zeta-weighting                 0.500000
      * Dispersion:
        s8                             2.700000
        a1                             0.520000
        a2                             5.000000
        s9                             5.000000
      * Repulsion:
        kExp                           1.500000    1.000000
        rExp                           1.000000
      * Coulomb:
        alpha                          2.000000
        third order                    shell-resolved
        anisotropic                    true
        a3                             3.000000
        a5                             4.000000
        cn-shift                       1.200000
        cn-exp                         4.000000
        max-rad                        5.000000

q/qsh data taken from xtbrestart
CAMM data taken from xtbrestart

          ...................................................
          :                      SETUP                      :
          :.................................................:
          :  # basis functions                   4          :
          :  # atomic orbitals                   4          :
          :  # shells                            2          :
          :  # electrons                         6          :
          :  max. iterations                   250          :
          :  Hamiltonian                  GFN2-xTB          :
          :  restarted?                       true          :
          :  GBSA solvation                  false          :
          :  PC potential                    false          :
          :  electronic temp.          300.0000000     K    :
          :  accuracy                    1.0000000          :
          :  -> integral cutoff          0.2500000E+02      :
          :  -> integral neglect         0.1000000E-07      :
          :  -> SCF convergence          0.1000000E-05 Eh   :
          :  -> wf. convergence          0.1000000E-03 e    :
          :  Broyden damping             0.4000000          :
          ...................................................

 iter      E             dE          RMSdq      gap      omega  full diag
   1     -3.7694211 -0.376942E+01  0.840E-15    0.00       0.0  T
   2     -3.7694211  0.000000E+00  0.672E-15    0.00       0.0  T
   3     -3.7694211  0.000000E+00  0.517E-15    0.00       0.0  T

   *** convergence criteria satisfied after 3 iterations ***

         #    Occupation            Energy/Eh            Energy/eV
      -------------------------------------------------------------
         1        2.0000           -0.7434383             -20.2300
         2        1.3333           -0.5697291             -15.5031
         3        1.3333           -0.5697291             -15.5031 (HOMO)
         4        1.3333           -0.5697291             -15.5031 (LUMO)
      -------------------------------------------------------------
                  HL-Gap            0.0000000 Eh            0.0000 eV
             Fermi-level           -0.5690705 Eh          -15.4852 eV

 SCC (total)                   0 d,  0 h,  0 min,  0.038 sec
 SCC setup                      ...        0 min,  0.004 sec ( 11.262%)
 Dispersion                     ...        0 min,  0.011 sec ( 28.682%)
 classical contributions        ...        0 min,  0.006 sec ( 16.028%)
 integral evaluation            ...        0 min,  0.015 sec ( 40.037%)
 iterations                     ...        0 min,  0.001 sec (  2.924%)
 molecular gradient             ...        0 min,  0.000 sec (  0.730%)
 printout                       ...        0 min,  0.000 sec (  0.229%)

         :::::::::::::::::::::::::::::::::::::::::::::::::::::
         ::                     SUMMARY                     ::
         :::::::::::::::::::::::::::::::::::::::::::::::::::::
         :: total energy              -3.769421097051 Eh    ::
         :: gradient norm              0.000000000000 Eh/a0 ::
         :: HOMO-LUMO gap              0.000000000000 eV    ::
         ::.................................................::
         :: SCC energy                -3.769421097051 Eh    ::
         :: -> isotropic ES            0.000000000000 Eh    ::
         :: -> anisotropic ES          0.000000000000 Eh    ::
         :: -> anisotropic XC         -0.000000000000 Eh    ::
         :: -> dispersion              0.000000000000 Eh    ::
         :: repulsion energy           0.000000000000 Eh    ::
         :: add. restraining           0.000000000000 Eh    ::
         :: total charge               0.000000000000 e     ::
         :::::::::::::::::::::::::::::::::::::::::::::::::::::

      ----------------------------------------------------------- 
     |                   =====================                   |
     |                        A N C O P T                        |
     |                   =====================                   |
     |               Approximate Normal Coordinate               |
     |                Rational Function Optimizer                |
      ----------------------------------------------------------- 
================
 final structure:
================
1
 xtb: 6.6.1 (8d0f1dd)
O            0.00000000000000        0.00000000000000        0.00000000000000

 Bond Distances (Angstroems)
 ---------------------------


 selected bond angles (degree)
 --------------------


 selected dihedral angles (degree)
 ---------------------------------

           -------------------------------------------------
          |                Final Singlepoint                |
           -------------------------------------------------

          ...................................................
          :                      SETUP                      :
          :.................................................:
          :  # basis functions                   4          :
          :  # atomic orbitals                   4          :
          :  # shells                            2          :
          :  # electrons                         6          :
          :  max. iterations                   250          :
          :  Hamiltonian                  GFN2-xTB          :
          :  restarted?                      false          :
          :  GBSA solvation                  false          :
          :  PC potential                    false          :
          :  electronic temp.          300.0000000     K    :
          :  accuracy                    1.0000000          :
          :  -> integral cutoff          0.2500000E+02      :
          :  -> integral neglect         0.1000000E-07      :
          :  -> SCF convergence          0.1000000E-05 Eh   :
          :  -> wf. convergence          0.1000000E-03 e    :
          :  Broyden damping             0.4000000          :
          ...................................................

 iter      E             dE          RMSdq      gap      omega  full diag
   1     -3.7694211 -0.376942E+01  0.414E-15    0.00       0.0  T
   2     -3.7694211  0.000000E+00  0.331E-15    0.00       0.0  T
   3     -3.7694211  0.000000E+00  0.255E-15    0.00       0.0  T

   *** convergence criteria satisfied after 3 iterations ***

         #    Occupation            Energy/Eh            Energy/eV
      -------------------------------------------------------------
         1        2.0000           -0.7434383             -20.2300
         2        1.3333           -0.5697291             -15.5031
         3        1.3333           -0.5697291             -15.5031 (HOMO)
         4        1.3333           -0.5697291             -15.5031 (LUMO)
      -------------------------------------------------------------
                  HL-Gap            0.0000000 Eh            0.0000 eV
             Fermi-level           -0.5690705 Eh          -15.4852 eV

 SCC (total)                   0 d,  0 h,  0 min,  0.001 sec
 SCC setup                      ...        0 min,  0.001 sec ( 42.254%)
 Dispersion                     ...        0 min,  0.000 sec (  1.472%)
 classical contributions        ...        0 min,  0.000 sec (  1.033%)
 integral evaluation            ...        0 min,  0.000 sec (  7.831%)
 iterations                     ...        0 min,  0.000 sec ( 33.769%)
 molecular gradient             ...        0 min,  0.000 sec (  6.507%)
 printout                       ...        0 min,  0.000 sec (  5.070%)

         :::::::::::::::::::::::::::::::::::::::::::::::::::::
         ::                     SUMMARY                     ::
         :::::::::::::::::::::::::::::::::::::::::::::::::::::
         :: total energy              -3.769421097051 Eh    ::
         :: gradient norm              0.000000000000 Eh/a0 ::
         :: HOMO-LUMO gap              0.000000000000 eV    ::
         ::.................................................::
         :: SCC energy                -3.769421097051 Eh    ::
         :: -> isotropic ES            0.000000000000 Eh    ::
         :: -> anisotropic ES          0.000000000000 Eh    ::
         :: -> anisotropic XC         -0.000000000000 Eh    ::
         :: -> dispersion              0.000000000000 Eh    ::
         :: repulsion energy           0.000000000000 Eh    ::
         :: add. restraining           0.000000000000 Eh    ::
         :: total charge               0.000000000000 e     ::
         :::::::::::::::::::::::::::::::::::::::::::::::::::::

           -------------------------------------------------
          |                Property Printout                |
           -------------------------------------------------

    * Orbital Energies and Occupations

         #    Occupation            Energy/Eh            Energy/eV
      -------------------------------------------------------------
         1        2.0000           -0.7434383             -20.2300
         2        1.3333           -0.5697291             -15.5031
         3        1.3333           -0.5697291             -15.5031 (HOMO)
         4        1.3333           -0.5697291             -15.5031 (LUMO)
      -------------------------------------------------------------
                  HL-Gap            0.0000000 Eh            0.0000 eV
             Fermi-level           -0.5690705 Eh          -15.4852 eV

     #   Z          covCN         q      C6AA      α(0)
     1   8 O        0.000     0.000    15.568     5.197

 Mol. C6AA /au·bohr⁶  :         15.568032
 Mol. C8AA /au·bohr⁸  :        314.171359
 Mol. α(0) /au        :          5.196904


Wiberg/Mayer (AO) data.
largest (>0.10) Wiberg bond orders for each atom

 ---------------------------------------------------------------------------
     #   Z sym  total        # sym  WBO       # sym  WBO       # sym  WBO
 ---------------------------------------------------------------------------
     1   8 O    0.000
 ---------------------------------------------------------------------------


molecular dipole:
                 x           y           z       tot (Debye)
 q only:        0.000       0.000       0.000
   full:        0.000       0.000       0.000       0.000
molecular quadrupole (traceless):
                xx          xy          yy          xz          yz          zz
 q only:        0.000       0.000       0.000       0.000       0.000       0.000
  q+dip:        0.000       0.000       0.000       0.000       0.000       0.000
   full:        0.000       0.000       0.000       0.000       0.000       0.000

           -------------------------------------------------
          |                Geometry Summary                 |
           -------------------------------------------------

      molecular mass/u    :       15.9994049
   center of mass at/Å    :        0.0000000       0.0000000       0.0000000
  moments of inertia/u·Å² :        0.0000000E+00   0.0000000E+00   0.0000000E+00
rotational constants/cm⁻¹ :             Infinity        Infinity        Infinity


optimized geometry written to: xtbopt.xyz


           -------------------------------------------------
          | TOTAL ENERGY               -3.769421097051 Eh   |
          | GRADIENT NORM               0.000000000000 Eh/α |
          | HOMO-LUMO GAP               0.000000000000 eV   |
           -------------------------------------------------

------------------------------------------------------------------------
 * finished run on 2024/10/14 at 13:29:59.319     
------------------------------------------------------------------------
 total:
 * wall-time:     0 d,  0 h,  0 min,  0.140 sec
 *  cpu-time:     0 d,  0 h,  0 min,  0.442 sec
 * ratio c/w:     3.164 speedup
 SCF:
 * wall-time:     0 d,  0 h,  0 min,  0.038 sec
 *  cpu-time:     0 d,  0 h,  0 min,  0.147 sec
 * ratio c/w:     3.855 speedup
 ANC optimizer:
 * wall-time:     0 d,  0 h,  0 min,  0.005 sec
 *  cpu-time:     0 d,  0 h,  0 min,  0.009 sec
 * ratio c/w:     1.797 speedup

